Source/Core/Core/HW/GCPadEmu.cpp

```diff
@@ -50,7 +50,10 @@ GCPad::GCPad(const unsigned int index) : m_index(index)
   // buttons
   groups.emplace_back(m_buttons = new ControllerEmu::Buttons(_trans("Buttons")));
   for (unsigned int i = 0; i < sizeof(named_buttons) / sizeof(*named_buttons); ++i)
-    m_buttons->controls.emplace_back(new ControllerEmu::Input(named_buttons[i]));
+  {
+    const std::string& ui_name = (named_buttons[i] == "Start") ? "START" : named_buttons[i];
```

Comment:
I don't think it should be necessary to note the reason that the string was changed. Code comments are not a changelog, and other changed config names like "Sideways Wiimote" and "Main Stick" don't have any comments like this.